src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -60,21 +53,7 @@
 import org.opensearch.action.support.master.AcknowledgedResponse;
 import org.opensearch.action.update.UpdateRequest;
 import org.opensearch.action.update.UpdateResponse;
-import org.opensearch.client.Cancellable;
-import org.opensearch.client.Client;
-import org.opensearch.client.ClusterAdminClient;
-import org.opensearch.client.ClusterClient;
-import org.opensearch.client.GetAliasesResponse;
-import org.opensearch.client.IndicesClient;
-import org.opensearch.client.Node;
-import org.opensearch.client.RequestOptions;
-import org.opensearch.client.Requests;
-import org.opensearch.client.RestClient;
-import org.opensearch.client.Request;
-import org.opensearch.client.Response;
-import org.opensearch.client.ResponseListener;
-import org.opensearch.client.RestClientBuilder;
-import org.opensearch.client.RestHighLevelClient;
+import org.opensearch.client.*;
```

Comment:
Can you take care of the wildcard import?

Reply:
But I like stars! :) 

![image](https://github.com/opensearch-project/opensearch-sdk-java/assets/9291703/58099c4c-16b0-44e2-87bc-a1d17b80b1a3)

Reply:
I swear everytime I forget to run `./gradlew spotlessApply` intelli J sneaks in this `*` imports